csi/volumeutils.py

```diff
@@ -427,6 +427,7 @@ def mount_glusterfs(volume, target_path):
     cmd = [
         GLUSTERFS_CMD,
         "--process-name", "fuse",
+        "-l", "/dev/stdout",
```

Comment:
stdout of this child process is not consumed.

Reply:
'stdout' (ie, fd=1) is not consumed, but here, glusterfs opens `-l /dev/stdout` as any other file. So, it would be writing to stdout device even faster